controller/lifecycle/lifecycle.go

```diff
@@ -330,7 +336,7 @@ func (l *LifecycleManager) reconcileSubroutine(ctx context.Context, instance Run
 	}
 
 	if err != nil {
-		if err.Sentry() {
+		if generationIsDifferent && err.Sentry() {
 			log.Error().Err(err.Err()).Msg("subroutine ended with error")
```

Comment:
🟢  Is that error logging duplicated (l.340 vs l.343) - remove?

Reply:
yes, removed.